app/services/availability_service.py

```
@@ -217,3 +222,74 @@ def _log_errors(self, results: list[RoomAvailability | Exception], date_context:
+                    
+                    price = self.pricing_service.calculate_total_price(
+                        base_price=room.pricePerHour,
+                        price_config=room.priceConfig or [],
+                        base_capacity=room.baseCapacity,
+                        extra_charge=room.extraCharge,
+                        start_dt=start_dt,
+                        end_dt=end_dt,
+                        people_count=request.capacity
+                    )
+                    res.estimated_price = price
+                except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)